### PR TITLE
Add trino server type

### PR DESCRIPTION
**Pull request by stefannegele**
https://github.com/datacontract/datacontract-cli/issues/264

### CODE REVIEW

**Comment by jochenchrist** [comment]
LGTM

**Comment by jochenchrist** [comment]
Can you add Trino to the README as well?

File: README.md [hunk 1]
```diff
@@ -508,6 +508,16 @@ servers:
 | region | `string` | AWS region, e.g., `eu-west-1`.                                            |
 | format | `string` | The format of the records. Examples: json, avro, protobuf.                |
 
+#### Postgres Server Object
```
Comment (jochenchrist) [inline README.md:511]: Headline Postgres -> Trino

Comment (stefannegele) [reply README.md]: Fixed

**Pull request merged**